doc/source/howtos/virt_sandbox.rst

```diff
@@ -340,22 +420,23 @@ To confirm that the user has been set up correctly, you can try to log in:
 
    Type 'help;' or '\h' for help. Type '\c' to clear the current input statement.
 
-To provide read/write access to the cluster for ProxySQL, add this user on one
+To provide read/write access to the cluster for |proxysql|, add this user on one
```

Comment:
We need to add application user creation at the beginning of `Creating ProxySQL Client User`. Once we have the application user in PXC we can proceed with user creation in ProxySQL DB.